src/App.js

```diff
@@ -1,24 +1,74 @@
-import logo from './logo.svg';
 import './App.css';
+import React, { useState } from 'react'; 
+import PokemonList from './components/PokemonList/PokemonList'
+import Header from './components/Header/Header'
+import Cart from './components/Cart/Cart'
+import Order from './components/Order/Order'
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
 
 function App() {
+  const [ cartItems, setCartItems ] = useState([]);
+  const [ order, setOrder ] = useState([]);
+
+  const onAdd = (pokemon) =>{
+    const exist = cartItems.find(item => item.id === pokemon.id);
+    if(exist){
+      setCartItems(cartItems.map(item => item.id === pokemon.id ? {...exist, cuantity: exist.cuantity + 1} : item));
```

Comment:
You probably meant `quantity` and not `cuantity`

Reply:
Yes indeed! hahaha, that spanglish of mine